### PR TITLE
Adding CyRK Python Package (v0.11.0)

### PR DESCRIPTION
**Pull request by jrenaud90**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jrenaud90** [comment]
Hey @conda-forge/help-python-c,
Here is an initial PR for adding CyRK to conda-forge. There are a few issues that I would like some help on when you have time:
- The linter is saying "Non noarch packages should have python requirement without any version constraints." Not sure what it is looking at / wanting. The package does have specific python version requirements. 
- The pipeline successfully built the package on Win and MacOS. For Linux it is failing but not at the build stage which passed. It throws a "raise OverLinkingError(overlinking_errors)" during the "compile .pyc files" step of packaging after successfully building. 

Thank you so much!

**Comment by jrenaud90** [comment]
@conda-forge-admin, please ping team @conda-forge/help-python-c,

**Comment by jrenaud90** [comment]
conda-forge/help-python-c

File: recipes/CyRK/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/cyrk-{{ version }}.tar.gz
+  sha256: 63e48e92b846195623ddd798921af8aeee7bfcbd034964b0ddbf96da8d6fc2e0
+
+build:
+  skip: true  # [py>312]
+  skip: true  # [py<38]
+  skip: true  # [py2k]
```
Comment (ocefpaf) [inline recipes/CyRK/meta.yaml:15]: These are not built on conda-forge. No need to skip them.

```suggestion
```

**Comment by ocefpaf** [comment]
I'm not sure how to fix the overlink errors here.

**Comment by jrenaud90** [comment]
Fixed some bugs in the newest version of the package and it appears to be passing tests now. Ready for next steps.

@conda-forge-admin, please ping team @conda-forge/help-python-c

**Pull request merged**